server/common/colors.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+HEX_COLOR_FORMAT = re.compile("^#[a-fA-F0-9]{6,6}$")
+
+
+def convert_color_to_hex_format(unknown):
+    if type(unknown) in (list, tuple):
```

Comment:
please add a doc string describing what this function accepts.

also, this file needs a reference to the data format spec - it seems to be anndata specific, so what does that entail?